0ldgranpas_Garage

NFTpinups opened the pull request:
https://twitter.com/CNFT_Pinups/status/1497424931281326081

https://twitter.com/CNFT_Pinups/

http://pinup.gallery

KaiNFT commented:
Dear Contributor, 

This request will now be closed, make a new ticket with the following criteria; the new file must be within the 'Projects' directory. Please check the README.md file for further information.

NFTpinups commented:
> Hey Grandpa, I had the same issue, was a bit frustrating because it isn't explicitly given as a necessary feature. Your project name must have no gaps or special characters in it for e.g. "Project" : "GrandpasOldGarage" I removed the pull request and deleted the failed version (just makes it less confusing) and redid it correctly. You can try editing the original pull request code but again it just made it messy and potentially confusing. GuruDingus

>Thank you very much for your helpful reply to this! :)
